### PR TITLE
Merge release/5.0 branches to master for Asp.Net

### PR DESCRIPTION
**Pull request by wtgodbe**
@mmitche @dotnet/aspdoi PTAL

### CODE REVIEW

**Comment by wtgodbe** [comment]
As it currently stands this only gets triggered when something gets merged into a trigger branch - if we don't rebuild preview1, we'll need to do a manual mop-up merge (as described in https://github.com/dotnet/aspnetcore-internal/pull/3533)

**Comment by wtgodbe** [comment]
This now matches any release/5.0 branch so we don't need to update it later. It won't do merges between 5.0-preview branches, though

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -1081,6 +1081,25 @@
         }
       }
     },
+    // Automate opening PRs to merge aspnet release/5.0-preview1 branches back to master.
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/aspnetcore/blob/release/5.0**//**/*",
```
Comment (mmitche) [inline Maestro/subscriptions.json]: Should just be release/5.0/**/*

**Comment by wtgodbe** [comment]
Matt pointed out that this thing could miss commits if the last commit to a previewN branch comes after a commit that conflicted with master - we may still need to do manual mop-ups, or get the bot to run on a schedule

**Comment by dougbu** [comment]
@dagood could you please get this in?

**Pull request merged**